denops/@denops-private/service.ts

```diff
@@ -126,11 +130,36 @@ class Plugin {
     try {
       const mod = await import(`${this.script}${suffix}`);
       await emit(this.#denops, `DenopsSystemPluginPre:${this.name}`);
+      globalThis.dispatchEvent(
+        new CustomEvent("denops:pluginloadstart", {
```

Comment:
I think it would be good to define a data type for the event. Use it like `new CustomEvent<PluginLoadStartDetail>`.